services/config/ffc-sfd.yaml

```diff
@@ -2,5 +2,8 @@ topics:
   - name: ffc-sfd-messages
     subscriptions:
       - ffc-sfd-messages-processor
+  - name: ffc-sfd-customer-receiver
+    subscriptions:
+      - ffc-sfd-customer-receiver-messages
```

Comment:
Discussed with SFD devs on naming of topics and subscription for customer receiver services and went with this format. The idea would be to build upon the subscriptions under the `ffc-sfd-customer-receiver` topic to include future services that we'll provision (e.g. `ffc-sfd-customer-receiver-applications`, `ffc-sfd-customer-receiver-payments` etc.). The Service Bus scripts for `ffc-sfd` would look like this with those future services included:
```
- name: ffc-sfd-customer-receiver
   subscriptions: 
       - ffc-sfd-customer-receiver-messages
       - ffc-sfd-customer-receiver-applications
       - ffc-sfd-customer-receiver-payments
```
And so on.